docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,78 @@
+      <th class="field-name">Date:</th>
+      <td class="field-body">December 7, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights 
+- **Percona Platform (Technology Preview):** 
+    - Ability to connect your Percona Monitoring and Management instances to your Percona Platform to get the latest intelligence from Percona and have all your accounts and relations with Percona centralized. READ_MORE
+
+- **Enhanced PostgreSQL monitoring**
+    - From this release, PMM will let users who add PostgreSQL Servers be monitored by PMM to [specify database name for monitoring](https://deploy-preview-643--pmm-doc.netlify.app/details/commands/pmm-admin.html#postgresql). Previous versions have the default `postgres` database, which wasn't always convenient for the users. 
+    - This Release of PMM is supported and compatible with the recent release of the PostgreSQL extension - `pg_stat_monitor` v1.0.0. You can read more about its features and advantages in this blog post (LINK to pg_stat_monitor release announcement or Userguide)
+    - PMM 2.25 also be compatible and working fine with the latest Percona Disctibutions for PostgreSQL released today (LINK), which includes the newest version of pg_stat_monitor extension.  
+
+- **Grafana usability enhancements**
+    - We changed the way how the dashboards, plugins, and data sources are added into PMM. We now use the native Grafana provisioning mechanism in PMM. This change will lead to a faster and more reliable update process for our users from one version of PMM to another. With this update, we'll preserve all custom changes you might already do with plugins and dashboards, but we'll encourage you to be back up them before an upgrade and recheck after, just in case. 
+    - This version of PMM also will let you change the time zone on dashboards. It will also preserve this selection while you navigate over Dashboards. If you need to change this setting permanently for your account - you need to change it in your preferences by the URL: https://YOUR_PMM_SERVER/graph/profile   
+
+- **DBaaS (Technical Preview)**
+    - PMM will now let you deploy and update your PXC clusters created with DBaaS functionality to the version of Percona Distribution for MySQL Operator 1.9.0 to deliver the latest fixes in this version.
+
+- **PMM environment enhancements**
+    - For users who use PMM in automation, we added a pmm-client docker container to be started as a sidecar container and not fail if the connection to DB  is not available. Please see full details in 

```

Comment:
Not sure I get why the "used in automation" is important here...

Reply:
🤔 .. in the usual case, you will start it manually and need to see - if the result was successful. No?